packages/compile-solidity/src/index.ts

```diff
@@ -191,8 +208,4 @@ const Compile = {
   }
 };
 
-module.exports = {
-  Compile,
-  CompilerSupplier,
-  Shims
-};
+export { Compile, CompilerSupplier };
```

Comment:
On further review, I don't think that this change in export syntax _does_ introduce a breaking change _here_. As such, it might be a better idea to delete the explicit `export` line and switch to using the `export` keyword inline with the definitions.

Please do take this as a nudge to review the export syntax in other modules (such as in `compile-solidity/dist/profiler`)

Reply:
okie doke